src/f5_tts/infer/utils_infer.py

```diff
@@ -5,33 +5,35 @@
 
 sys.path.append(f"../../{os.path.dirname(os.path.abspath(__file__))}/third_party/BigVGAN/")
 
+from f5_tts.model.utils import (
+    get_tokenizer,
+    convert_char_to_pinyin,
+)
+from f5_tts.model import CFM
+from vocos import Vocos
+from transformers import pipeline
+from pydub import AudioSegment, silence
+import tqdm
+import torchaudio
+import torch
+import numpy as np
+import matplotlib.pylab as plt
```

Comment:
`from xxx impor xxx` go after `import xxx`
and better in alphabet order of pkg name first letter
same for following codes